modules/tcia_dataset.ipynb

```diff
@@ -1,845 +1,845 @@
+    "from monai.transforms import (\n",
+    "    Activations,\n",
+    "    AsDiscrete,\n",
+    "    EnsureChannelFirstd,\n",
+    "    Compose,\n",
+    "    LoadImaged,\n",
+    "    Orientationd,\n",
+    "    RandCropByPosNegLabeld,\n",
+    "    RandScaleIntensityd,\n",
+    "    RandShiftIntensityd,\n",
+    "    RandFlipd,\n",
+    "    Spacingd,\n",
+    "    NormalizeIntensityd,\n",
+    "    ResampleToMatchd,\n",
+    ")\n",
+    "import monai\n",
+    "from monai.networks.nets import UNet\n",
+    "from monai.metrics import DiceMetric\n",
+    "from monai.losses import DiceCELoss\n",
+    "from monai.inferers import sliding_window_inference\n",
+    "from monai.apps import TciaDataset\n",
+    "from monai.apps.tcia import TCIA_LABEL_DICT\n",
+    "from monai.data import DataLoader, decollate_batch\n",
+    "import os\n",
+    "import requests\n",
+    "import tempfile\n",
+    "import shutil\n",
+    "import torch\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6mxYYFvPBTHw"
+   },
+   "source": [
+    "## Setup data directory\n",
+    "\n",
+    "You can specify a directory with the `MONAI_DATA_DIRECTORY` environment variable.  \n",
+    "This allows you to save results and reuse downloads.  \n",
+    "If not specified a temporary directory will be used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "wbTvWiEaBTHw",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "directory = os.environ.get(\"MONAI_DATA_DIRECTORY\")\n",
+    "root_dir = tempfile.mkdtemp() if directory is None else directory\n",
+    "print(root_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Y6mNIzYOBTHw"
+   },
+   "source": [
+    "## Brief Introduction\n",
+    "\n",
+    "Documentation about TCIA's REST APIs can be found at https://wiki.cancerimagingarchive.net/x/NIIiAQ. A full list of TCIA's Collections can be found at https://www.cancerimagingarchive.net/collections/. This is the best place to read abstracts about the available datasets and discover supporting data/labels (e.g. clinical spreadsheets and other non-DICOM data). The following code can also be run to get an idea of what each collection contains in terms of patient counts, modalities and body parts examined."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "2pplIY81iu94"
+   },
+   "outputs": [],
+   "source": [
+    "# list collections with patient counts, modalities and body parts examined\n",
+    "# running this can take ~5min to complete\n",
+    "\n",
+    "API_URL = \"https://services.cancerimagingarchive.net/nbia-api/services/v1/\"\n",
+    "\n",
+    "collection_url = API_URL + \"getCollectionValues\"\n",
+    "collection_data = requests.get(collection_url).json()\n",
+    "\n",
+    "for x in collection_data:\n",
+    "    collection_name = x['Collection']\n",
```

Comment:
mainly changed the variables here, for some reason all the indentations are modified by the jupyter notebook application